src/main/java/core/basesyntax/service/impl/CreateTheReportImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.TypeOfOperation;
+import core.basesyntax.service.CreateTheReport;
+import core.basesyntax.strategy.ActionStrategyImpl;
+import core.basesyntax.strategy.ActionsStrategy;
+import java.util.List;
+
+public class CreateTheReportImpl implements CreateTheReport {
+    private static final int INDEX_0 = 0;
+    private static final int INDEX_1 = 1;
+    private static final int INDEX_2 = 2;
```

Comment:
Not understandeble names. Better OPERETION_INDEX, FRUIT_INDEX, AMOUNT_INDEX  etc